torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -136,7 +136,6 @@ QuantFusionInfo getBinaryOpScalarFusionInfo(
   std::string op_pattern =
       getInputTensorQParamOpPattern(op_name, extra_op_args);
 
-  const auto& extra_op_arg_list = getExtraArgList(extra_op_args);
```

Comment:
isn't this used in the next line?

Reply:
I requested review before checking tests and then immediately unrequested, sorry about that. Let me figure out what's going on. This dead store was flagged by an internal tool.

Reply:
maybe it's related to `const auto&`, you can try changing to `auto`